test/integration/BUILD

```diff
@@ -417,6 +417,7 @@ envoy_sh_test(
     size = "large",
     srcs = ["run_envoy_test.sh"],
     cc_binary = [":hotrestart_main"],
+    coverage = False,
```

Comment:
i think you can remove this now - it should be fixed by bazelrc flag

Reply:
It was failing just recently, which PR fixes it? the binary is too large to be linked with coverage.

Reply:
fixed here https://github.com/envoyproxy/envoy/pull/31641

i just fixed it

Reply:
OK, giving it a go. It is hovering close to 2GB linker limit - basically every shell test that needs an envoy binary with extensions is very close to failing to compile with coverage.

Reply:
Well, it doesn't pass locally. It's not about grpc, something in abseil just grew the binary larger (probably some deep method got bigger) and now it passed 2GB linker size. I think we need to remove these "whole binary" shell tests from coverage, but that will lower `main()` coverage.

Reply:
cc @alyssawilk FYI.

Reply:
i hit this updating absl - didnt figure out the solution so it blocked update

i hit it again when updating grpc - this time i added the bazel flag - and it is fixed (at least to the best of our understanding/internet searching)

cc @ravenblackx

Reply:
Coverage fails https://dev.azure.com/cncf/envoy/_build/results?buildId=159097&view=logs&j=4930ecaf-18f4-5b3c-dea3-309729c3b3ae&t=573d8780-d7b9-52e3-b4e0-a89886b0b9ff. Will reapply the skip coverage and lower the coverage numbers.

Reply:
@ravenblackx seems like this isnt the same issue after all - this is the other issue we looked at `relocation R_X86_64_PC32 `

Reply:
to be more precise and avoid confusion - the fix applied resolved the initial `relocation` error - but then it hit this 2nd one 

apologies for noise